package.json

```diff
@@ -54,6 +54,7 @@
     "tempfile": "^1.1.1",
     "through2": "^2.0.3",
     "tmp": "^0.0.31",
+    "url": "^0.11.0",
```

Comment:
Since critical is for usage in node, why not use the [node core module](https://nodejs.org/api/url.html) instead of an npm?